.github/workflows/notify_on_commit_to_develop.yml

```diff
@@ -0,0 +1,34 @@
+name: Broadcast 'push to develop' event to our release-scripts repo
```

Comment:
Is this an expected change?

Reply:
@kevintab95 No I believe this is the same issue as above. I didn't make any changes to this file locally.